lib/android/app/src/test/java/com/reactnativenavigation/viewcontrollers/stack/StackControllerTest.kt

```diff
@@ -194,6 +194,15 @@ class StackControllerTest : BaseTest() {
         verify(child1, never()).mergeOptions(any())
     }
 
+    @Test
+    fun setRoot_activityDestroyDuringAnimationShouldNotCrash() {
```

Comment:
This test has no assertions and it seems like it's name doesn't describe the test correctly.